list.json

```diff
@@ -721,44 +581,44 @@
 				"BodyParts": {
 					"Head": {
 						"Health": {
-							"Current": 35,
-							"Maximum": 35
+							"Current": 70,
```

Comment:
This is the change in quesiton.

Reply:
oops totally forgot to revert normal health for the profile
I will update in a minute : )